src/helm/benchmark/run_specs/medcalc_bench_specs.py

```diff
@@ -0,0 +1,132 @@
+            # One way of doing so is having receiving the calculator ID in this function and passing it to
+            # the scenario, which can then filter the dataset samples by the calculator ID.
+            question=(
+                "What is the patient's Creatinine Clearance using the Cockroft-Gault Equation in terms of mL/min? "
+                "You should use the patient's adjusted body weight in kg instead of the patient's actual body "
+                "weight if the patient is overweight or obese based on their BMI. If the patient's BMI's normal, "
+                "set their adjusted body weight to the minimum of the ideal body and actual weight. If the "
+                "patient is underweight, please set their adjusted body weight to their actual body weight."
+            ),
+            calculator_id="2",
+        ),
+        input_noun=None,  # Set directly in the scenario.
+        output_noun="\n\nCalculated Value",
+        max_tokens=500,
+    )
+
+
+def _get_zero_shot_cot_instructions() -> str:
+    """Generate instructions for the MedCalc-Bench scenario.
+
+    This function is inspired on the system prompt definition in the original code:
+    https://github.com/ncbi-nlp/MedCalc-Bench/blob/048ba77dbe332e9190935e4a30965bff444b940e/evaluation/run.py#L16
+
+    Credits to the original authors: https://github.com/ncbi-nlp/MedCalc-Bench.
+    """
+
+    return (
+        "You are a helpful assistant for calculating a score for a given patient note. "
+        "Please think step-by-step to solve the question and then generate the required score. "
+        "Your output should contain the step by step thinking and the final answer, which is a short and direct answer to the question. "
+        'Before giving the final answer, write "Calculated Value: " followed by the answer.'
+    )
+
+
+def _get_one_shot_cot_instructions(question: str, calculator_id: str) -> str:
+    """Generate instructions for the MedCalc-Bench scenario.
+
+    This function is inspired on the system prompt definition in the original code:
+    https://github.com/ncbi-nlp/MedCalc-Bench/blob/048ba77dbe332e9190935e4a30965bff444b940e/evaluation/run.py#L26
+
+    Credits to the original authors: https://github.com/ncbi-nlp/MedCalc-Bench.
+
+    In the original code, there's exactly one example response for each calculator ID.
+    These examples are stored in a JSON file: https://github.com/ncbi-nlp/MedCalc-Bench/blob/048ba77dbe332e9190935e4a30965bff444b940e/evaluation/one_shot_finalized_explanation.json
+    None of the examples include the actual questions. They only contain the step-by-step thinking and the final answer.
+    Looking at the dataset samples we can see that all samples with the same calculator ID use the same question.
+    The original expect that for each sample, we collect the calculator ID and the question for building the one-shot instructions.
+    """
+    examples: Dict = {}
+    with open(ONE_SHOT_EXAMPLES_URL, "r") as f:
```

Comment:
If you move this inside the scenario as I suggested above, you should also replace this raw `open()` with `ensure_file_downloaded()` instead.